redis_enterprise/assets/configuration/spec.yaml

```diff
@@ -7,4 +7,20 @@ files:
     - template: init_config/default
   - template: instances
     options:
-    - template: instances/default
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.value.example: https://<host>:8070/metrics
+        openmetrics_endpoint.description: |
+          Insert Description here for rdse OpenMetrics endpoint or link to docs
+        openmetrics_endpoint.display_priority: 3
+        namespace.value.example: rdse
+        namespace.hidden: false
+        namespace.enabled: true
+        namespace.display_priority: 2
```

Comment:
```suggestion
```

I think these can/should be deleted. I don't think it's being used in the code since rdse is hardcoded to the check itself in `__NAMESPACE__`. This shouldn't ideally be customizable as it alters all metrics that this check would pickup if changed and will thus break all the dashboards etc.

Reply:
Understood, removed